server.go

```diff
@@ -31,6 +31,25 @@ type User interface {
 	CreateMessageLimit() *uint32
 }
 
+// A mailbox that supports retrieving per-mailbox message size limits.
+type Mailbox interface {
```

Comment:
This isn't used anywhere. We can probably just expect backends to use `StatusSetAppendLimit` when needed?

Reply:
Wops, sorry. Added almost mechanically without even thinking about usage.